components/elm/src/biogeophys/BalanceCheckMod.F90

```diff
@@ -344,7 +344,7 @@ subroutine ColWaterBalanceCheck( bounds, num_do_smb_c, filter_do_smb_c, &
        do fc = 1,num_do_smb_c
           c = filter_do_smb_c(fc)
           g = col_pp%gridcell(c)
-          write(iulog,*)'WARNING:  glc_dyn_runoff_routing = ', glc_dyn_runoff_routing(g)  ! TKT
+!          write(iulog,*)'WARNING:  glc_dyn_runoff_routing = ', glc_dyn_runoff_routing(g)  ! TKT
```

Comment:
should this be part of this merge?

Reply:
Yes, @stephenprice checked with @tktesfa2014 about this, and the determination is it was a debugging write that had been meant to be commented but was still active.  It was leading to thousands of write statements when running compsets with active MALI.  Normally we make a rule to not touch anything outside of the MALI directory in our fork, but given the simple nature of this change we included it in Steve's updates (https://github.com/MALI-Dev/E3SM/pull/16).